app/assets/stylesheets/application.css

```diff
@@ -1,5 +1,4 @@
 @charset "utf-8";
 
-@import "bourbon";
-@import "neat";
 @import "normalize.css/normalize.css";
+@import "budget.css";
```

Comment:
No need to the `.css` extension to be specified

Reply:
Done in ad-css-styling sha: 12530f8